sql/src/main/java/org/elasticsearch/action/admin/indices/create/TransportBulkCreateIndicesAction.java

```diff
@@ -643,16 +643,32 @@ protected ClusterBlockException checkBlock(BulkCreateIndicesRequest request, Clu
     }
 
     @Override
-    public void killAllCalled(long timestamp) {
+    public void killAllJobs(long timestamp) {
         lastKillAllEvent = timestamp;
+        // todo also provide with id
```

Comment:
why with id? the pendingOperation killing here was already working as it should?